README.md

```diff
@@ -46,15 +46,15 @@ How
 ### From command line
 
 ```sh
-npm-git-install
+npm-git-install install
```

Comment:
Actually, why not rename the CLI tool to `npm-git`? Then we would have `npm-git install`, `npm-git shrinkwrap` and possibly more in the future.